### ISSUE TITLE
`@ref` Directive to region from imported file broken in v5.0.3

### ISSUE THREAD

**Issue by fredrikhr** [AnWeber/httpyac#212]
### `f1.http`

```
###
# @name F1R1
{{
  module.exports = { f1r1v1: 'foo' };
}}
```

### `f2.http`

```
###
# @import ./f1.http
# @ref F1R1

###
# @name F2R1
GET http://example.org
```

### `f3.http`

```
### 
# @import ./f1.http
# @ref F1R1
# @import ./f2.http
# @ref F2R1

###
# @name F3R1
GET http://example.org
  /{{F2R1.some_property}}
X-SOME-HEADER: {{f1r1v1}}
```

# Expected behaviour

In region `F3R1` both `F2R1` and the exported value of `f1r1v1` should be defined.

# Actual behaviour

Neither `F2R1` nor `f1r1v1` is defined when executing/sending region `F3R1`.

As executed with the above files within a newly created docker container using the `mcr.microsoft.com/vscode/devcontainers/javascript-node:16` image.

```
$ npx httpyac@5.0.3 send --all --verbose f3.http
load core
load dotenv
load eventSource
load graphql
load grpc
load http
load intellij
load injection
load javascript
load mqtt
load oauth2
load rabbitMQ
load websocket
json parse of /package.json failed
{}
global
parse imported file /f1.http
json parse of /package.json failed
load core
load dotenv
load eventSource
load graphql
load grpc
load http
load intellij
load injection
load javascript
load mqtt
load oauth2
load rabbitMQ
load websocket
execute global scripts for import /f1.http
parse imported file /f2.http
json parse of /package.json failed
load core
load dotenv
load eventSource
load graphql
load grpc
load http
load intellij
load injection
load javascript
load mqtt
load oauth2
load rabbitMQ
load websocket
execute global scripts for import /f2.http
global
parse imported file /f1.http
F3R1
init request
send GET http://example.org/{{F2R1.some_property}}
replace variables in request
expression F2R1.name throws error
ReferenceError: F2R1 is not defined
    at Object.<anonymous> (/f3.http:11:20)
    at bt (/root/.npm/_npx/6878136aff7dfd08/node_modules/httpyac/dist/index.js:3:41565)
    at Vt (/root/.npm/_npx/6878136aff7dfd08/node_modules/httpyac/dist/index.js:3:42066)
    at Object.oi [as action] (/root/.npm/_npx/6878136aff7dfd08/node_modules/httpyac/dist/index.js:3:44911)
    at qt.<anonymous> (/root/.npm/_npx/6878136aff7dfd08/node_modules/hookpoint/dist/index.js:1:2752)
    at Generator.next (<anonymous>)
    at u (/root/.npm/_npx/6878136aff7dfd08/node_modules/hookpoint/dist/index.js:1:1262)
expression f1r1v1 throws error
ReferenceError: f1r1v1 is not defined
    at Object.<anonymous> (/f3.http:12:20)
    at bt (/root/.npm/_npx/6878136aff7dfd08/node_modules/httpyac/dist/index.js:3:41565)
    at Vt (/root/.npm/_npx/6878136aff7dfd08/node_modules/httpyac/dist/index.js:3:42066)
    at Object.oi [as action] (/root/.npm/_npx/6878136aff7dfd08/node_modules/httpyac/dist/index.js:3:44911)
    at qt.<anonymous> (/root/.npm/_npx/6878136aff7dfd08/node_modules/hookpoint/dist/index.js:1:2752)
    at Generator.next (<anonymous>)
    at u (/root/.npm/_npx/6878136aff7dfd08/node_modules/hookpoint/dist/index.js:1:1262)
request {
  decompress: true,
  retry: 0,
  throwHttpErrors: false,
  headers: {
    accept: '*/*',
    'user-agent': 'httpyac',
    'X-SOME-HEADER': '{{f1r1v1}}'
  },
  timeout: undefined,
  https: undefined,
  http2: undefined,
  cookieJar: CookieJar {
    rejectPublicSuffixes: true,
    enableLooseMode: false,
    allowSpecialUseDomain: false,
    store: { idx: {} },
    prefixSecurity: 'silent',
    _cloneSync: [Function (anonymous)],
    _importCookiesSync: [Function (anonymous)],
    getCookiesSync: [Function (anonymous)],
    getCookieStringSync: [Function (anonymous)],
    getSetCookieStringsSync: [Function (anonymous)],
    removeAllCookiesSync: [Function (anonymous)],
    setCookieSync: [Function (anonymous)],
    serializeSync: [Function (anonymous)]
  },
  method: 'GET'
}

---------------------

GET http://example.org/%7B%7BF2R1.name%7D%7D
accept: */*
accept-encoding: gzip, deflate, br
user-agent: httpyac
x-some-header: {{f1r1v1}}

HTTP/1.1 404  - Not Found
cache-control: max-age=604800
connection: close
content-encoding: gzip
content-length: 648
content-type: text/html; charset=UTF-8
date: Mon, 28 Feb 2022 14:26:58 GMT
expires: Mon, 07 Mar 2022 14:26:58 GMT
server: EOS (vny/0451)
vary: Accept-Encoding

```

We can see that neither `F2R1` nor `f1r1v1` is defined. This used to be the case in version v4.10.

**Comment by AnWeber** [comment]
The reason is that f1.http was imported twice. Here the processing was aborted by mistake. Workaround is to import `f1.http` only once.

**Comment by AnWeber** [comment]
published v5.0.4
